RuslanImankulov/homework1.js

```diff
@@ -0,0 +1,61 @@
+//Func to check palindrome
```

Comment:
From my point of view palindrome check should be case insensitive, but your approach is ok also.